extensions/kafka/src/main/java/com/hazelcast/jet/kafka/KafkaSources.java

```diff
@@ -62,10 +62,9 @@ private KafkaSources() {
      * The source creates a {@code KafkaConsumer} for each {@link Processor}
      * instance using the supplied {@code properties}. It assigns a subset of
      * Kafka partitions to each of them using manual partition assignment (it
-     * ignores the {@code group.id} property). Default local parallelism for
-     * this processor is 2 (or less if less CPUs are available). The Kafka's
-     * message timestamp will be used as a {@linkplain
-     * StreamSourceStage#withNativeTimestamps(long) native timestamp}.
+     * ignores the {@code group.id} property).
```

Comment:
Please reflow the paragraph, the rule is 72 chars per line (not counting initial indentation or `*`).

Reply:
FIxed this now. The following line is 74 characters for the sake of readability:

    will be used as a {@linkplain StreamSourceStage#withNativeTimestamps(long)

There are several lines with more than 72 characteres so I thought it is ok.
Let me know if this is still unacceptable.

Reply:
Yeah, ideally those would be improved as well :) But this is fine.